pkg/install/1-installresources.go

```diff
@@ -668,9 +651,20 @@ func (i *Installer) installResources(ctx context.Context) error {
 			return err
 		}
 	}
-
 	{
-		ipAddress := lbIP.String()
+		var ipAddress string
+		{
+			// Discover what private IP address was assigned to internal LB
+			lbClient := mgmtnetwork.NewLoadBalancersClient(i.env.SubscriptionID())
+			lbClient.Authorizer = i.fpAuthorizer
```

Comment:
I somehow think this is the wrong authorizer. Did you try with one from cluster credentials?

Reply:
I will

Reply:
ignore :)